### PR TITLE
[canary-release]: 2.0.0-canary.1 - Data Table documentation

### PR DESCRIPTION
**Pull request by jd239**
## Summary
This PR is to release the new Data Table documentation to gh-pages. The Data Table codebase itself has not been amended.
The publish should be triggered when merged. 

## Checklist

- [x] Commit subject includes component name (i.e. feat(canary-web-component): [ic-<component-name>] ....)
- [x] Component tag has been updated with `unstable`. (i.e. `ic-**unstable**-<component_name>`).
- [x] Updated type references to new component tag where relevant (i.e. `IcUnstable<Component_Name>`).
- [x] `npm run build` has been re-run.
- [x] Component has a story.
- [x] Component has been accessibility tested.
- [x] Component has unit and e2e tests.

### CODE REVIEW

File: packages/react/src/stories/ic-data-table.stories.mdx [hunk 1]
```diff
@@ -81,1735 +128,687 @@ export const basicData = [
...
+```
+
+### Column overrides
+
+There may be a requirement to style a column differently to the rest of the data to add emphasis. For example, centre align data within the 'first name' column. To do this, add `columnAlignment` to the column. `columnAlignment` takes an object which sets the `horizontal` and `vertical` property.
+
+- `horizontal` accepts `left`, `right` or `center`.
+- `vertical` accepts `top`, `middle`, `bottom`.
+
+`emphasis` can be added to the column by setting `high` or `low`
```
Comment (ad9242) [inline packages/react/src/stories/ic-data-table.stories.mdx:636]:
```suggestion
`emphasis` can be added to the column by setting `high` or `low`.
```

File: packages/react/src/stories/ic-data-table.stories.mdx [hunk 2]
```diff
@@ -81,1735 +128,687 @@ export const basicData = [
...
+    data={data}
+  />
+);
+
+export default DataTable;
+```
+
+### Row Overrides
+
+Row overrides allow styling to be applied at a row level and is set within the `header` object. The row overrides are used inconjunction with row headers. The following row overrides options apply:
```
Comment (ad9242) [inline packages/react/src/stories/ic-data-table.stories.mdx:691]:
```suggestion
Row overrides allow styling to be applied at a row level and is set within the `header` object. The row overrides are used in conjunction with row headers. The following row overrides options apply:
```

**Review by ad9242 — CHANGES_REQUESTED** [review]
same comments apply to web components ic-data-table.stories.mdx

File: packages/web-components/src/components/ic-data-table/ic-data-table.stories.mdx [hunk 3]
```diff
@@ -5,1974 +5,661 @@ import {
...
+Use the data table component to display data in a tabular form.
+
+There are two required props for data table:
+
+- Caption: `string`
+- Columns: `IcUnstableDataTableColumnObject[]`
+
+Click the 'Component API' tab to view all the available props, events and slots for Data Tables.
+
+To import the data table web component into your application, import `@ukic/canary-web-components` into your application.
```
Comment (GCHQ-Developer-299) [inline packages/web-components/src/components/ic-data-table/ic-data-table.stories.mdx:48]: The previous comment about wording in the react storybook would apply here too

File: packages/web-components/src/components/ic-navigation-group/ic-navigation-group.css [hunk 4]
```diff
@@ -166,6 +166,7 @@
   transition: var(--ic-easing-transition-slow);
   overflow: hidden;
 }
+
```
Comment (GCHQ-Developer-299) [inline packages/web-components/src/components/ic-navigation-group/ic-navigation-group.css:169]: Is this a necessary change?

File: packages/web-components/src/components/ic-data-table/ic-data-table.stories.mdx (hunk 3 above)
Comment (jd239) [reply packages/web-components/src/components/ic-data-table/ic-data-table.stories.mdx:48]: I've changed it to 'To use the data table component.... The term `web component` has been removed from both making it more generic.

File: packages/web-components/src/components/ic-navigation-group/ic-navigation-group.css (hunk 4 above)
Comment (jd239) [reply packages/web-components/src/components/ic-navigation-group/ic-navigation-group.css:169]: This is just to improve the files readability. New block of CSS so have a empty line to separate it.

**Comment by evenstensberg** [comment]
@GCHQ-Developer-299 Shouldn't this package be archived if the same/main development work is in ic-ui-kit?

**Comment by GCHQ-Developer-299** [comment]
@evenstensberg i'll raise it with the team, i think you're right - we migrated the canary changes into the main ic-ui-kit ages ago (if this PR is any indication, 14 months! )

**Pull request closed without merging**